Resources/Prototypes/Actions/spells.yml

```diff
@@ -0,0 +1,11 @@
+﻿- type: action
+  actionType: Fireball
+  icon: Interface/Alerts/Human/human1.png
+  name: "Fireball"
+  filters:
+    - human
+  description: "This is a [color=red]no-wiznerd fireball[/color]."
+  behaviorType: TargetPoint
+  behavior: !type:ProjectileSpell
+   castmessage: EL NATH!
```

Comment:
Cast message for fireball is `ONI SOMA`.
(Plus it's `EI` with an i and not an L)